Marlin/src/pins/stm32/pins_BIGTREE_SKR_PRO_V1.1.h

```diff
@@ -186,7 +186,7 @@
 const int SPI_Pins [NUM_SPI_BUSES][5] = {
 // MOSI, MISO, SCK , SS  , SW
   {PB15, PB14, PB13, PB12, PF12}, //EXT2 (Probed first. The card in the display it's probably easier to reach.)
-  {PB5 , PA6 , PA5 , PA4 , PB11}, //TF
+  {PA7 , PA6 , PA5 , PA4 , PB11}, //TF
```

Comment:
Seems to be plausible. D13 at PA7 is flashing during firmware updates.
(If we can still have any trust in the SCH)

Reply:
No. @akselii did with me some continuity tests. It's definitely pb12.
Probably pa7 flashes because they instructed the firmware to.
When I will have the board here I will do all the measurements myself.